activiti-steps/activiti-steps-assertions-api/src/main/java/org/activiti/steps/operations/TaskOperations.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.steps.operations;
+
+import org.activiti.api.task.model.payloads.ClaimTaskPayload;
+import org.activiti.api.task.model.payloads.CompleteTaskPayload;
+import org.activiti.steps.assertions.TaskAssertions;
+
+public interface TaskOperations {
+
+    TaskAssertions claim(ClaimTaskPayload claimTaskPayload);
+
+    TaskAssertions complete(CompleteTaskPayload completeTaskPayload);
```

Comment:
suspend and delete are missing here

Reply:
Yes, these operations are not covered yet.